content/platform/data-services-cloud-console/home.md

```diff
@@ -1,177 +1,42 @@
 ---
 title: Data Services Cloud Console
-version: 0.1.1
+version: 0.2.1
 description: Easy to read guides, tips and documentation to rapidly help any
   developer working with Data Services Cloud Console APIs.
 image: /img/platforms/image-1-dscc.svg
 width: large
 priority: 1
 tags:
   - data-services-cloud-console
-  - greenlake-common-cloud
+  - greenlake-cloud-console
 ---
-Customers across industries are struggling with the complexity of managing data and infrastructure, because it creates a roadblock to innovation and agility. Today, every organization is required to unleash the power of data to drive digital transformation, but fragmented data management tools, manual processes, and infrastructure silos - spanning edge to cloud - are getting in the way. This complexity also amplifies business risk, and it's only getting harder as data continues to grow, apps evolve, and infrastructure continues its spread from edge to cloud.
+To eliminate the silos and complexity of data management and infrastructure, HPE provides the Data Services Cloud Console (DSCC), a SaaS based cloud console that delivers cloud operational agility and unified data operations as a service. Data Services Cloud Console also offers a unified API that gives developers access to infrastructure and data as code. 
 
-![Intelligent Data Platform](/img/dscc-idp-core-architect.png "DSCC Diagram")
+![DSCC UI](/img/a-single-destination-for-data-and-infra-services.png "DSCC UI diagram")
 
-Data Services Cloud Console public REST API provides a resource for customers who are looking to enhance their infrastructure management and data-ops using the programmatical extensions from Data Services Cloud Console. Data Services Cloud Console (DSCC) brings the cloud experience to wherever data lives and streamlines data management across your hybrid cloud. It provides a suite of cloud services across your edge, core, and cloud to accelerate data, agility, and innovation for everyone, from data managers to data innovators.
+Data Services Cloud Console (DSCC) brings the cloud experience to wherever data lives and streamlines data management across your hybrid cloud. It provides a suite of cloud services across your edge, core, and cloud to accelerate data, agility, and innovation for everyone, from data managers to data innovators. Delivered as a SaaS based cloud console, DSCC moved customer’s innovation faster because there is nothing to deploy, always updated to the latest software features, and reduce risk of ownership by subscription. The console is a single destination for any data and infrastructure-services, that comprises of both the cloud data services and cloud infrastructure services.  
```

Comment:
Slight modifications required to dark green highlighted text:
Place a hyphen between "SaaS" and "based" so it reads "Delivered as a SaaS-based cloud console..."
Second part of that sentence needs to be rephrased to "...DSCC speeds customer innovation because there is nothing to deploy, everything is always updated with the latest software, and ownership risk is reduced by virtue of it being a subscription."
The last sentence is very confusing. Do you mean that "The console is a single management point for any data or infrastructure service that are found in the cloud data services and cloud infrastructure services catalog."? Single destination throws me, as does the reitteration of data services and infrastructure services.